packages/esm-patient-list-app/src/api/api-remote.ts

```diff
@@ -15,36 +15,42 @@ import {
 
 export const cohortUrl = '/ws/rest/v1/cohortm';
 
-async function postData(url: string, data = {}, ac = new AbortController()) {
```

Comment:
I'd keep the `AbortController` parameter. `AbortController`s are used to signal to `fetch()` that the request can be cast aside because the component waiting for the data no longer needs it. The most obvious use-case for this is something like:

1. Component mounts and requests data
2. While data is being fetched, the component is unmounted due to user action

At 2, we want to stop the data request (or at least signal that its unimportant), but that means having an `AbortController` that's aware of the lifecycle of the containing component (which is kind of the point of the `useAbortController()` hook I added to the framework).

Basically, a purely local-variable `AbortController()` is just overhead, since it will never abort.

Reply:
Yeah, that makes a lot of sense. I'll revert back to the previous behaviour.